core/tests/protractor/explorationFeedback.js

```diff
@@ -0,0 +1,87 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for feedback on explorations
+ * Tests the following sequence:
+ * User 1 creates and publishes an exploration.
+ * User 2 plays the exploration and leaves feedback on it
+ * User 1 reads the feedback and responds to it.
+ * Note: In production, after this sequence of events, the only notification
+ * User 2 will receive is via email, and we can't easily test this
+ * in an e2e test.
+ */
+
+var dashboard = require('../protractor_utils/dashboard.js');
+var editor = require('../protractor_utils/editor.js');
+var general = require('../protractor_utils/general.js');
+var library = require('../protractor_utils/library.js');
+var player = require('../protractor_utils/player.js');
+var users = require('../protractor_utils/users.js');
+var workflow = require('../protractor_utils/workflow.js');
+
+describe('ExplorationFeedback', function() {
+  var EXPLORATION_TITLE = 'Sample Exploration';
+  var EXPLORATION_OBJECTIVE = 'To explore something';
+  var EXPLORATION_CATEGORY = 'Random';
+  var EXPLORATION_LANGUAGE = 'English';
+
+  beforeEach(function() {
+    users.createUser('user1@ExplorationFeedback.com',
+                     'user1ExplorationFeedback');
+    users.createUser('user2@ExplorationFeedback.com',
+                     'user2ExplorationFeedback');
+  });
+
+  it('adds feedback to an exploration', function() {
+    var feedback = 'A good exploration. Would love to see a few more questions';
+    var feedbackResponse = 'Thanks for the feedback';
+
+    // Creator creates and publishes an exploration
+    users.login('user1@ExplorationFeedback.com');
```

Comment:
FYI I think creator@ExplorationFeedback.com is acceptable (with username creatorExplorationFeedback) and it does have the benefit of being more understandable. Ditto for the learner. I was mainly intending to caution against @gmail.com.